Upgrade Community Calls/Call_001.md

```diff
@@ -0,0 +1,271 @@
+
+**Ben Edgington**
+* Ben, So, so Danny, I, that's covers the credential change message. I think we agree to use the Genesis four version, but the withdrawal message remains signed with the fork version, right? So that expires The exit. 
+
+**Danny**
+* Yeah, and I, I know the terminology is kind of clear. Sorry, the exit Unclear. Yes, you're correct, Yeah, but no, but, but you're right, the message I was talking about wasn't a withdrawal message as well. So there are exits and there are key changes. 
+* If you're exited and you have your, on your key change, you will be withdrawn. So, so yes, Ben, the, the exit, which is actually maybe a more akin to withdrawal message, does degrade over those two epochs, and I don't suspect that's something that is changed, although it's been discussed, it won't be for this fork. it's something that's kind of been discussed lately here and there. 
+
+**Tim**
+* Okay. okay, another question about the, the two cues, and this is something that's like comes up, I get a DM about this probably every day. so can you please cover what order the withdrawal queue and sweeps are processed? So I think basically the, I think the interaction of the exit queue and the withdrawal sweep, we probably need like some sort of animation for this, but Danny, do you wanna Yeah, Right. Yeah, expand on That. 
+
+**Danny**
+* So there's an activation queue, you are active, then you submit a voluntary exit and then you enter into the exit queue. The exit queue is, you are still active while you're in the exit queue, you still have duties you can still make, you can still, gain and lose rewards based off of your attestations and proposed performance.
+*  On the other side of the exit queue, you are no longer active, but you are not yet withdrawable. There is the men validator withdrawal ability delay. It's 256 epochs, that's about 27 hours. So you're out of the exit queue, you are exited, you then have a 27 hour delay and now you are with Drawable At that point. 
+* You are part of the, you can be part of the withdrawal sweep, which is round robin, so you know, if the last, if you're a validator 32 and on the last block it left off at 31, then you're the next one in the week if you're validator 32. And the, last block it left off was at 64, you're almost at the end of this week is gonna go all the way to the end and then back at the beginning. So exit Queue with all ability delay, then round robins sweep clear as day, right? Yeah. 
+
+**Tim**
+* Someone says we should call it limbo when you're exited waiting  to be withdrawn. 
+
+**Ben Edgington**
+* Yeah, it's not, it's not very long limbo. Right? I mean, even if every, even if every validate has 0 0 1 credentials, which they don't, at the moment only about, 40% due,  it's will still take I think four days to complete a whole sweep through the entire validator set. 
```

Comment:
```suggestion
* Yeah, it's not, it's not very long limbo. Right? I mean, even if every, even if every validate has 0x01 credentials, which they don't, at the moment only about, 40% due, it will still take I think four days to complete a whole sweep through the entire validator set. 
```